src/Relude/Monad/Maybe.hs

```diff
@@ -55,7 +55,11 @@ whenJust (Just x) f = f x
 whenJust Nothing _  = pass
 {-# INLINE whenJust #-}
 
--- | Monadic version of 'whenJust'.
+{- | Monadic version of 'whenJust'.
+
+TODO: whenJustM ??
```

Comment:
You can provide example as for `whenJust`, just wrap the value in `pure`. And add some text that this function is useful if you want to work with functions that return values of type `IO (Maybe a)` or something like that.

Btw, the same documentation can be added to `whenLeftM` and `whenRightM` and `whenNothingM` :slightly_smiling_face: